master/reference/calicoctl/resources/policy.md

```diff
@@ -47,14 +50,15 @@ spec:
 
 #### Spec
 
-| Field      | Description                                                                                                                                           | Accepted Values | Schema                | Default |
-|------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|-----------------------|---------|
-| order      | (Optional) Indicates priority of this policy, with lower order taking precedence.  No value indicates highest order (lowest precedence)               |                 | float                 |         |
-| selector   | Selects the endpoints to which this policy applies.                                                                                                   |                 | [selector](#selector) | all()   |
-| ingress    | Ordered list of ingress rules applied by policy.                                                                                                      |                 | List of [Rule](#rule) |         |
-| egress     | Ordered list of egress rules applied by this policy.                                                                                                  |                 | List of [Rule](#rule) |         |
-| doNotTrack | Indicates to apply the rules in this policy before any data plane connection tracking, and that packets allowed by these rules should not be tracked. | true, false     | boolean               | false   |
-| preDNAT    | Indicates to apply the rules in this policy before any DNAT.                                                                                          | true, false     | boolean               | false   |
+| Field      | Description                                                                                                                                           | Accepted Values | Schema                | Default  |
+|------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|-----------------------|----------|
+| order      | Indicates ordering of this policy: policies with lower order are applied earlier. No value means that the policy is applied last.                     |                 | float                 |          |
```

Comment:
What if there is more than one policy with no value?

Reply:
most recently created policy is hit first.

Reply:
I don't think that's right - I believe policies with the same order are enforced lexicographically by name.

Reply:
Just expanded my experiment to adding an additional policy ("1deny") and Casey's right - lexicographically by name.

Reply:
Awesome, thanks @bcreane and @caseydavenport ! So then, which in the lexicographic order gets applied first?